webapp/src/Controller/Jury/SubmissionController.php

```diff
@@ -147,10 +200,10 @@ public function indexAction(
             'submissions' => $submissions,
             'submissionCounts' => $submissionCounts,
             'showContest' => count($contests) > 1,
-            'hasFilters' => !empty($filters),
+            'hasFilters' => $hasFilters,
             'results' => $results,
             'showExternalResult' => $this->dj->shadowMode(),
-            'showTestcases' => count($submissions) <= $latestCount,
+            'showTestcases' => true,
```

Comment:
Is this used anywhere else?

Reply:
We only show it on the general submission page and not others (like team page, judgehost page, etc.). So yes.